mainwindow.cpp

```diff
@@ -73,7 +73,7 @@ MainWindow::MainWindow(QWidget *parent)
 
 
     // check database version (Debug)
-    int db_ver = aDB->dbVersion();
+    int db_ver = aDB->version();
```

Comment:
We could call the database 'version' revision to more clearly differentiate between the program and the database. When reading database version it makes me think of the sqlite version as in `ADatabase::sqliteVersion()`

How about this?

`ADatabase::sqliteVersion()` becomes `ADatabase::version()`
`ADatabase::version()` becomes `ADatabes::revision()`

 With the docstrings `\brief` outlining what you get from each.

Reply:
yeah... i forgot about that. In that case lets just go with dbVersion. the only problem is that the variable and the getter conflict in name. Ill keep the variable databaseVersion to make it verbose since its not going to be typed on its own asidfe from connect and ill keep the function dbVersion